src/behaviors/Column.js

```diff
@@ -9,7 +9,8 @@ var propertyNames = [
     'name',
     'header',
     'calculator',
-    'type'
+    'type',
+    'halign'
```

Comment:
Not here. Rather on the column properties object, which automatically defers to the grid properties object before deferring to the theme properties object before deferring (finally) to the default properties object.